_shared/hooks/post_gen_project.py

```diff
@@ -160,7 +160,6 @@ def main():
             "package.json",
             "yarn.lock",
             ".docker.env",
-            "{{ cookiecutter.package_name }}/pshell.py",
```

Comment:
Remove `pshell.py` from the cookiecutter's `template_ignore_patterns`, meaning that the `make template` commands in projects will now update `pshell.py` from the latest version of the template. I think we do want the cookiecutter to own `pshell.py`: I think we probably want this file to be the same in every project. Projects can always add `pshell.py` to their own per-project ignore lists in `cookiecutter.py` if they want to do their own thing